app/build.gradle

```diff
@@ -11,7 +11,7 @@ android {
 
     defaultConfig {
         applicationId "com.cmput301f20t21.bookfriends"
-        minSdkVersion 24
```

Comment:
where do we need this change? just curious tho.

Reply:
String.join() lol

Reply:
For joining author name when a book has multiple authors